src/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Code from OutboundLink module in gatsby-plugin-google-analytics
+ * modified for gtag
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function OutboundLink(props) {
+  return (
+    <a
+      {...props}
+      onClick={e => {
+        if (typeof props.onClick === `function`) {
+          props.onClick();
+        }
+        let redirect = true;
+        if (
+          e.button !== 0 ||
+          e.altKey ||
+          e.ctrlKey ||
+          e.metaKey ||
+          e.shiftKey ||
+          e.defaultPrevented
+        ) {
+          redirect = false;
+        }
+        if (props.target && props.target.toLowerCase() !== `_self`) {
+          redirect = false;
+        }
+
+        if (window.gtag) {
+          window.gtag('event', 'click', {
+            event_category: 'outbound',
+            event_label: props.href,
+            transport_type: 'beacon',
```

Comment:
In the original `OutboundLink` transport depends on whether it is a redirect or not. Should that also be the case here? https://github.com/gatsbyjs/gatsby/blob/45ec012f03a533c28b4ebf3b944f2342a7bd86f2/packages/gatsby-plugin-google-analytics/src/index.js#L31

Reply:
@flpvsk That's what's generally shared for anyone tracking outbound links. Based on this reference to the old analytics.js api, I suspect having beacon set is the norm: "in the future, analytics.js will likely switch to using 'beacon' as the default mechanism in browsers that support it." Overall it's kind of vague. Thoughts?

Reply:
@ascasson alright, I agree. Makes sense to keep it as the default.